openbas-front/src/admin/components/atomic_testings/InjectList.tsx

```diff
@@ -190,17 +181,13 @@ const InjectList: FunctionComponent<Props> = ({
         </ListItem>
         {injects.map((injectDto) => {
           return (
-
             <ListItem
               key={injectDto.inject_id}
               classes={{ root: classes.item }}
               divider
               secondaryAction={
                 <AtomicTestingPopover
                   atomic={injectDto}
-                  entries={[{ label: 'Duplicate', action: () => handleOpenDuplicate(injectDto.inject_id) }]}
```

Comment:
Why this changes.
Now we have the possibility to update & delete in the list.
This is not the behavior we want:
- adding delete should be handle in this issue https://github.com/OpenBAS-Platform/openbas/issues/1205
- update is not possible in the list direclty

Reply:
I'll make the changes! ty